### ISSUE TITLE
Q: Do votes on PRs from general public count?

### ISSUE THREAD

**Issue by TomasMikula** [non/algebra#56]
The [development process](https://github.com/non/algebra#development-process) doesn't say that only +1s from people with merge access count. Does it mean that votes from anyone count?

**Comment by non** [comment]
I'm not sure there is consensus on this point. But in my opinion, I find outside +1s very useful in an advisory way. If there were many such +1s, or the person giving the +1 was a domain expert (or asked for their opinion) I might consider it as "counting" in a process sense.

I do value +1s from everyone, so in that sense I think they all count.

**Comment by TomasMikula** [comment]
Thanks, this answers my question. Not sure whether you want to keep this open until there is such a consensus on the issue or the content of your answer is included in the README.

**Comment by non** [comment]
I'd like to hear from @johnynek, @tixxit, @avibryant, and @larsrh on this point. Then I'll update the README accordingly and close this. Thanks for asking, it is an important question.

**Comment by johnynek** [comment]
I agree that reviews are always helpful. I think it would be useful to have some way of joining the group of people for whom +1 counts (something like having N PRs merged or support of a quorum of current members), but I think the quorum/consensus thing from a pre-published set is best (especially if, perhaps inevitably, a PR comes that is controversial).

**Comment by sritchie** [comment]
Agree here - shall we close?
